js/tests/mocks.spec.ts

```diff
@@ -0,0 +1,14 @@
+import {describe, expect ,test} from 'vitest';
+import {getGeneratedMocks} from '../main.ts';
+import {Photo, PhotoComment} from '../contracts/common.ts';
+
+describe('equality of unique ids', () => {
+	const amount = 25;
+	const getUniqueIds = (arr: Photo[] | PhotoComment[]): number => new Set(arr.flatMap((el) => el.id)).size;
```

Comment:
Можно пикнуть нужное свойство
```suggestion
	const getUniqueIds = (arr: Pick<Photo, 'id'>[]): number => new Set(arr.map(({id}) => id)).size;
```
Почему flatMap я не понял